toolkit/docs/building/building.md

```diff
@@ -1,25 +1,24 @@
 # Building
-
-- [Building](#building)
-  - [Quick Start](#quick-start)
-    - [Prerequisites](#prerequisites)
-    - [Configure the Environment](#configure-the-environment)
-    - [ISO Quick Start](#iso-quick-start)
-    - [VHDX Quick Start](#vhdx-quick-start)
-    - [Build from Sources Quick Start](#build-from-sources-quick-start)
-  - [Further Reading](#further-reading)
-    - [Building in Stages](#building-in-stages)
-      - [1) Bootstrap Toolchain](#1-bootstrap-toolchain)
-        - [Full toolchain Bootstrap From Sources](#full-toolchain-bootstrap-from-sources)
-      - [2) Build All Packages](#2-build-all-packages)
-      - [3) Build Images](#3-build-images)
-    - [ISOs](#isos)
-    - [Packages](#packages)
-      - [Working on Packages](#working-on-packages)
-        - [DOWNLOAD_SRPMS](#download_srpms)
-        - [Force Rebuilds](#force-rebuilds)
-        - [Ignoring Packages](#ignoring-packages)
-        - [Source Hashes](#source-hashes)
+- [Overview](#Overview)
+- [Building in Stages](#building-in-stages)
+   - [Install Prerequisites](#install-prerequisites)
+   - [Clone and Sync To Stable Commit](#clone-and-sync-to-stable-commit)
+   - [Toolchain Stage](#toolchain-stage)
+     - [Populate Toolchain](#populate-toolchain)
+     - [Rebuild Toolchain](#rebuild-toolchain)
+   - [Package Stage](#package-stage)  
+     - [Rebuild All Packages](#rebuild-all-packages)
+     - [Rebuild Minimal Required Packages](#rebuild-minimal-required-packages)
+   - [Image Stage](#image-stage)
+     - [Virtual Hard Disks and Containers](#virtual-hard-disks-and-containers)
+     - [ISO Images](#iso_images)
```

Comment:
>#iso_images [](start = 20, length = 11)

Another broken link. I think you've used accidentally used an underscore instead of a hyphen.

Reply:
fixed.